src/unix/thread.c

```diff
@@ -510,7 +510,7 @@ typedef struct uv_semaphore_s {
   unsigned int value;
 } uv_semaphore_t;
 
-#if defined(__GLIBC__) || platform_needs_custom_semaphore
+#if (defined(__GLIBC__) && !defined(__UCLIBC__)) || platform_needs_custom_semaphore
```

Comment:
Long line. Can you wrap at the `||`?

Reply:
Do you mean to wrap the code like this?

```c
#if (defined(__GLIBC__) && !defined(__UCLIBC__)) || \
    platform_needs_custom_semaphore
...

#endif
```

I think this would make `platform_needs_custom_semaphore` look not relevant to `#if (defined(__GLIBC__) && !defined(__UCLIBC__))`, so I prefer to write them into one line. How do you think about it?

Reply:
That's what I mean, yes. It's idiomatic for the code base (there's a fairly strict 80 column limit) so please change it.

Reply:
I see. I have made the change.
Thank you.